apps/files_sharing/js/sharedialogview.js

```diff
@@ -0,0 +1,59 @@
+/**
+ * @author Jan Ackermann <jackermann@owncloud.com>
+ *
+ * @copyright Copyright (c) 2021, ownCloud GmbH
+ * @license GPL-2.0
```

Comment:
```suggestion
 * @license AGPL-3.0
```

GPL not AGPL? (https://github.com/owncloud/core/blob/master/COPYING)